packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -466,7 +466,7 @@ module.exports.subscriptionProductMetadataBaseValidator = isA
     successActionButtonURL: isA.string().uri().required(),
     appStoreLink: isA.string().uri().optional(),
     playStoreLink: isA.string().uri().optional(),
-    productSet: isA.string().optional(),
+    productSet: isA.string().required(),
```

Comment:
Do you know why productSet is a string in some places, and an array of strings in others?

Reply:
Per our conversation, productSet validates as a string (comma separated list) within metadata, and as an array of strings after the metadata is converted into an array for product configuration.